test/helpers/setup_polly.js

```diff
@@ -1,9 +1,10 @@
 /* eslint-disable import/no-extraneous-dependencies */
-import { resolve } from 'path';
-import FSPersister from '@pollyjs/persister-fs';
 import NodeHttpAdapter from '@pollyjs/adapter-node-http';
 import { Polly, setupMocha as setupPolly } from '@pollyjs/core';
-import { encodeCassetteResponseBodies, decodeCassetteResponseBodies } from './cassette_encoding';
+import FSPersister from '@pollyjs/persister-fs';
+import { resolve } from 'path';
+
```

Comment:
extra lina 
```suggestion
```

Reply:
These are intended and automatic due to an import sorting plugin, kind of like `isort` for Python